src/pages/Distinction/DistinctionTable/FormEditDistinctionTypes.tsx

```diff
@@ -153,7 +154,7 @@ const FormEditDistinctionTypes: React.FC<FormEditDistinctionTypesProps> = () =>
           {visRule ?
               <div>
                 <Text type="danger">
-                  Поле не повинно містити більше 250 символів!
+                  Поле не повинно містити більше 200 символів!
```

Comment:
please, set here the variable 'typeMaxLength' except of number 200

Reply:
ok, done